### ISSUE TITLE
A query gets a parsing error with LIMIT statement

### ISSUE THREAD

**Issue by tdhung80** [orientechnologies/orientdb#5589]
Hi,

I am migrating from version 1.7.8 to version 2.1.8, the SQL below does not working any more

```SQL
SELECT eval('$TotalListsQuery[0].Count') AS TotalLists
   LET $TotalListsQuery = ( SELECT Count(1) AS Count FROM ContactList WHERE Account=#20:1 AND EntityInfo.State=0)
 LIMIT 1
```

It is working if remove "LIMIT 1"

My Best,
Hung Tran

**Comment by luigidellaquila** [comment]
Hi @tdhung80 

We are checking it, I'll let you know soon

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @tdhung80 

I fixed the problem in both 2.1.x and develop branches. The fix will be released with 2.1.10

Thanks

Luigi
